umbraco-cms-webapp-redis-cache/azuredeploy.json

```diff
@@ -0,0 +1,525 @@
+         }
+      },
+      "serverName":{
+         "type":"string",
+         "defaultValue":"umbracocmsserver",
+         "metadata":{
+            "description":"SQL Azure DB Server name"
+         }
+      },      
+      "administratorLogin":{
+         "type":"string",
+         "metadata":{
+            "description":"SQL Azure DB administrator  user login"
+         }
+      },
+      "administratorLoginPassword":{
+         "type":"securestring",
+         "metadata":{
+            "description":"Database admin user password"
+         }
+      },
+      "databaseName":{
+         "type":"string",
+         "defaultValue":"umbracocmsdb",
+         "metadata":{
+            "description":"Database Name"
+         }
+      },
+      "edition":{
+         "type":"string",
+         "allowedValues":[
+            "Standard",
+            "Premium"
+         ],
+         "defaultValue":"Standard",
+         "metadata":{
+            "description":"SQL DB Sku : Standard,Premium"
+         }
+      },
+      "storageName":{
+         "type":"string",
+         "metadata":{
+            "description":"Storage Account Name"
+         }
+      },
+      "storageAccountType":{
+         "type":"string",
+         "allowedValues":[
+            "Standard-LRS",
+            "Standard-GRS",
+            "Standard-RAGRS",
+            "Standard-ZRS"
+         ],
+         "metadata":{
+            "description":"Storage Account Type : Standard-LRS, Standard-GRS,Standard-RAGRS,Standard-ZRS"
+         },
+         "defaultValue":"Standard-GRS"
+      },
+      "redisCacheName":{
+         "type":"string",
+         "metadata":{
+            "description":"Redis Cache Name"
+         }
+      },
+      "redisCacheSKU":{
+         "type":"string",
+         "allowedValues":[
+            "Basic",
+            "Standard"
+         ],
+         "metadata":{
+            "description":"Redis Cache Sku - Basic , Standard"
+         },
+         "defaultValue":"Standard"
+      },
+      "redisCacheCapacity":{
+         "type":"int",
+         "allowedValues":[
+            0,
+            1,
+            2,
+            3,
+            4,
+            5,
+            6
+         ],
+         "metadata":{
+            "description":"Redis Cache Capacity  , supported values from 0 to 6 represnting cache tier name C0 to C6 "
+         },
+         "defaultValue":0
+      },
+      "NonAdminDatabaseUsername":{
+         "type":"string",
+         "metadata":{
+            "description":"Non-admin Database User. Must be Unique"
+         }
+      },
+      "NonAdminDatabasePassword":{
+         "type":"securestring",
+         "metadata":{
+            "description":"Non-admin Database User password"
+         }
+      }
+   },
+   "resources":[
+      {
+         "apiVersion":"2014-04-01-preview",
+         "name":"[parameters('redisCacheName')]",
+         "type":"Microsoft.Cache/Redis",
+         "location":"[resourceGroup().location]",
```

Comment:
A point to consider when using resourceGroup().location is if the individual resources are available in all regions as ARM. If not then getting the location is better to avoid deployment failures.